pyresample/ewa/__init__.py

```diff
@@ -117,8 +117,8 @@ def ll2cr(swath_def, area_def, fill=np.nan, copy=True):
     ch = -abs(area_def.pixel_size_y)
     w = area_def.x_size
     h = area_def.y_size
-    ox = area_def.area_extent[0]
-    oy = area_def.area_extent[3]
+    ox = area_def.area_extent[0] + cw / 2.
```

Comment:
`area_extent[0]` is also the upper left x coordinate. Either way, you are on the left most extent of the grid in the X direction so you need to go right one half pixel. However, the line below that is incorrect now that I look at it because `ch` is negative so I'm subtracting a negative; going up when I mean to be going down "into" the grid.